data_model/src/events/mod.rs

```diff
@@ -39,17 +39,11 @@ impl VersionedEventPublisherMessage {
     }
 }
 
-/// Message sent by the stream producer
+/// Message sent by the stream producer.
+/// Event sent by the peer.
 #[version_with_scale(n = 1, versioned = "VersionedEventPublisherMessage")]
-#[derive(Debug, Clone, Decode, Encode, FromVariant, IntoSchema)]
-pub enum EventPublisherMessage {
-    /// Reply sent by the peer.
-    /// The message means that event stream connection is initialized and will be supplying
-    /// events starting with the next message.
-    SubscriptionAccepted,
-    /// Event sent by the peer.
-    Event(Event),
-}
+#[derive(Debug, Clone, Decode, Encode, IntoSchema)]
+pub struct EventPublisherMessage(pub Event);
```

Comment:
consider renaming to `EventMessage` or `Event`

Reply:
i will stick with `EventMessage`